### PR TITLE
Fix typos and style inconsistencies related to configuration

### PR DESCRIPTION
**Pull request by r6e**
## Description
This PR contains a handful of changes that are likely too small to warrant separate pull requests. They are as follows:

* `resources/profiles/Kingroon/process/*`: These configs do not have a `type` field. While this is inherited from their associated `fdm_process_common`, all other preset configs define this in the individual JSON files.
* Other `*.json` files: There are some characters left in these that resulted in technically-invalid JSON.
* `textured_plate_temp_initial_layer` has its `max` defined twice. It's clear that one of these was supposed to be a `min`.
* `machine_max_speed_` tooltip text is inconsistent with `machine_max_acceleration_` and `machine_max_jerk_` tooltips.
* `machine_max_jerk_` sidetext has the unit listed as `mm/s`, but "jerk" being the 3rd time-derivative of position is actually `mm/s³`.
* `retract_lift_above` and `retract_lift_below` are defined twice, likely the result of a stale branch when c97a880 was merged.

## Motivation and Context
I noticed these issues when working on putting together some documentation around the JSON configuration files ([see here](https://r6e.github.io/orcaslicer/configuration.html) for the WIP docs). This change is in the interest of consistency and coding/style standards, and also because these sorts of things don't often get much love in open-source projects.

Most of the changes introduced here are stylistic, and should not affect functionality (although it may be possible that some of the configs didn't load).

### CODE REVIEW

File: src/libslic3r/PrintConfig.cpp [hunk 1]
```diff
@@ -2495,11 +2495,11 @@ def = this->add("filament_loading_speed", coFloats);
             (void)L("Maximum speed E");
             def->category = L("Machine limits");
             def->readonly = false;
-            def->tooltip  = (boost::format("Maximum speed of %1% axis") % axis_upper).str();
-            (void)L("Maximum X speed");
-            (void)L("Maximum Y speed");
-            (void)L("Maximum Z speed");
-            (void)L("Maximum E speed");
+            def->tooltip  = (boost::format("Maximum speed of the %1% axis") % axis_upper).str();
+            (void)L("Maximum speed of the X axis");
```
Comment (SoftFever) [inline src/libslic3r/PrintConfig.cpp:2499]: The original text looks fine to me.
Can we revert the changes so that we won't loose all the translations?

File: src/libslic3r/PrintConfig.cpp [hunk 2]
```diff
@@ -2536,7 +2536,7 @@ def = this->add("filament_loading_speed", coFloats);
             (void)L("Maximum jerk of the Y axis");
             (void)L("Maximum jerk of the Z axis");
             (void)L("Maximum jerk of the E axis");
-            def->sidetext = L("mm/s");
+            def->sidetext = L("mm/s³");
```
Comment (SoftFever) [inline src/libslic3r/PrintConfig.cpp:2539]: it should me mm/s

File: src/libslic3r/PrintConfig.cpp [hunk 3]
```diff
@@ -3073,20 +3073,6 @@ def = this->add("filament_loading_speed", coFloats);
     def->mode = comAdvanced;
     def->set_default_value(new ConfigOptionEnumsGeneric{ ZHopType::zhtNormal });
 
-    def = this->add("retract_lift_above", coFloats);
```
Comment (SoftFever) [inline src/libslic3r/PrintConfig.cpp:3076]: Nice catch.
Can we remove the other one since these descriptions are well translated into other languages by various contributors already?

**Comment by SoftFever** [comment]
> Motivation and Context
> I noticed these issues when working on putting together some documentation around the JSON configuration files ([see here](https://r6e.github.io/orcaslicer/configuration.html) for the WIP docs). This change is in the interest of consistency and coding/style standards, and also because these sorts of things don't often get much love in open-source projects.
> 
> Most of the changes introduced here are stylistic, and should not affect functionality (although it may be possible that some of the configs didn't load).


This fantastic!

File: src/libslic3r/PrintConfig.cpp (hunk 2 above)
Comment (r6e) [reply src/libslic3r/PrintConfig.cpp:2539]: Doesn't the term "jerk" in this context refer to the [physics concept](https://en.wikipedia.org/wiki/Jerk_%28physics%29) of change in acceleration over time? If so, that's definitely mm/s³. If not, I'd be really interested to know what it represents, because that's how I've been thinking about it.

That being said, my intent here is clarity, so either way if the change is less-clear, then I'd be happy to remove it.

Comment (SoftFever) [reply src/libslic3r/PrintConfig.cpp:2539]: Well, it's slightly different in 3d printer.

Quoted from Marlin firmware:
>  Jerk is the maximum change in velocity (in mm/sec) that can occur instantaneously. 

In Klipper, it equals to SCV(Square corner velocity).
So the unit is mm/s

Comment (r6e) [reply src/libslic3r/PrintConfig.cpp]: That's good to know! Fixed.

**Comment by r6e** [comment]
@SoftFever I've made all the requested changes.

Also, I'd like to say that I really like OrcaSlicer, so thank you for developing/maintaining it!

**Review by SoftFever — APPROVED** [review]
Looks good.

**Comment by SoftFever** [comment]
Thank you.
Merged
